master/internal/core.go

```diff
@@ -382,6 +382,9 @@ func (m *Master) getAggregatedResourceAllocation(c echo.Context) error {
 		if err = writeAggType("username", entry.ByUsername); err != nil {
 			return err
 		}
+		if err = writeAggType("user_id", entry.ByUserId); err != nil {
```

Comment:
assuming this part works the code changes lgtm

Reply:
actually it doesn't work -- i was using `map<int32, float>` for `by_user_id` in `master.proto`, and `map[string]float32` is expected in `core.go`. decided to switch that to `map<string, float>`, please lmk if that doesn't seem like a good solution.